src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -245,14 +247,11 @@ export function RenderNickname ({ agentId, setCounterpartyNick, setCounterpartyF
   if (loading) {
     // TODO: Unsubscribe from Loader to avoid any potential mem leak.
     return <>
-      <Loader
+      <Loading
```

Comment:
again, I'm confused why these changes are appearing in this PR when they're already on develop. /shrug

Reply:
Hm... That is strange. :thinking: